### DISCUSSION TITLE
Disable Nginx Container

### DISCUSSION THREAD

**Discussion by Karbust** [goharbor/harbor#15517] [Q&A] [answered]
Hello,

Is it possible to disable the installation of the nginx container? I already have an nginx instance for other stuff, I don't want to have 2...

Thank you

**Comment by wy65701436** [comment, marked as answer]
How did you install Harbor? theoretically, it can as actually core is serving the 8080(by default) port of the configured hostname . But, you have to refer the nginx settings and update yours.
